workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -510,6 +511,19 @@ def get_ml_config(model_path):
     return config
 
 
+def get_emulation_config():
```

Comment:
Moved copy pasted integration test back into the main file.  Use monkeypatch to temporarily update environment variables.